packages/web/src/common/store/lineup/sagas.js

```diff
@@ -53,6 +55,9 @@ function* filterDeletes(tracksMetadata, removeDeleted) {
   const isUSDCGatedContentEnabled = yield getFeatureEnabled(
     FeatureFlags.USDC_PURCHASES
   )
+  const allowedHandles = yield remoteConfig
```

Comment:
maybe of no substance, but id think there'd be parentheses like `(yield remoteConfig.getRemoteVar(StringKeys.EXPLORE_PREMIUM_ALLOWED_USERS))?.split(',')`

actually, it's probably that `remoteConfig.getRemoteVar` does not need a yield